install/intro.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: e443388d4249fea383d6ec2d1a338be220cf95ee Maintainer: lucalzetta Status: ready -->
+<!-- $Revision$ -->
+  <chapter xml:id="install.general" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>Considerazioni generali sull'installazione</title>
+   <para>
+    Prima di iniziare l'installazione è necessario sapere che uso si farà 
+    di PHP. Ci sono tre settori principali in cui PHP trova applicazione 
+    come descritto nella sezione 
+    <link linkend="intro-whatcando">cosa può fare PHP?</link>
+    section:
+    <itemizedlist>
+     <listitem><simpara>Siti web e applicazioni web (programmi lato server)</simpara></listitem>
+     <listitem><simpara>Scripting da riga di comando</simpara></listitem>
+     <listitem><simpara>Applicazioni desktop (GUI)</simpara></listitem>
+    </itemizedlist>
+   </para>
+   <para>
+    Per la prima e più comune forma, sono necessarie tre cose: 
+    PHP, un server web e un browser web, probabilmente avrete già un 
+    web browser, e, conformemente alle impostazioni del vostro 
+    sistema operativo, potreste avere anche un server web (es. 
+    Apache su Linux e macOS o IIS su Windows). Potete anche aver 
+    noleggiato dello spazio web da un fornitore. In questo modo 
+    non avrete bisogno di impostare niente da parte vostra, solo 
+    scrivere i vostri script PHP, caricarli sul server che avete 
+    affittato e vedere i risultati sul vostro browser. 
```

Comment:
Ho notato che in questo file ci sono delle organizzazioni che vanno a spostare il contenuto rispetto a quello originale del file inglese, usando delle righe differenti. Questo ovviamente non è un problema in alcune situazioni al limite, ma nella maggior parte dei casi ha senso rispettare lo stesso ordine della versione inglese, in modo da semplificare quanto più possibile la manutenzione del file in futuro. Prendo come esempio questa sezione. Nella versione inglese è scritta così:

```xml
   <para>
    For the first and most common form, you need three things:
    PHP itself, a web server and a web browser. You
    probably already have a web browser, and depending on
    your operating system setup, you may also have a web
    server (e.g. Apache on Linux and macOS; IIS on Windows).
    You may also rent webspace at a company. This way, you
    don't need to set up anything on your own, only write
    your PHP scripts, upload it to the server you rent, and
    see the results in your browser. 
   </para>
```

nella tua invece abbiamo:

```xml
   <para>
    Per la prima e più comune forma, sono necessarie tre cose: 
    PHP, un server web e un browser web, probabilmente avrete già un 
    web browser, e, conformemente alle impostazioni del vostro 
    sistema operativo, potreste avere anche un server web (es. 
    Apache su Linux e macOS o IIS su Windows). Potete anche aver 
    noleggiato dello spazio web da un fornitore. In questo modo 
    non avrete bisogno di impostare niente da parte vostra, solo 
    scrivere i vostri script PHP, caricarli sul server che avete 
    affittato e vedere i risultati sul vostro browser. 
   </para>
```

Come la scriverei io, in modo da renderla più facile da modificare in futuro:

```xml
   <para>
    Per la prima e più comune forma, sono necessarie tre cose: 
    PHP, un server web e un browser web.
    Probabilmente avrete già un web browser, e, conformemente alle
    impostazioni del vostro sistema operativo, potreste avere anche un server
    web (es. Apache su Linux e macOS o IIS su Windows).
    Potete anche aver noleggiato dello spazio web da un fornitore. In questo modo 
    non avrete bisogno di impostare niente da parte vostra, solo scrivere
    i vostri script PHP, caricarli sul server che avete affittato e
    vedere i risultati sul vostro browser. 
   </para>
```

Quando ti parlavo di situazioni al limite, facevo riferimento al caso delle parole "web server" scritte su due righe differenti nella versione inglese ("web" su una e "server" sulla successiva). Nella traduzione, dovendo usare "server web", invertiamo l'ordine, portando "server" su una riga e "web" sulla sucessiva.